autotest.pm

```diff
@@ -32,7 +32,7 @@ our @testorder;    # for keeping them in order
 our $isotovideo;
 
 sub loadtest {
```

Comment:
I guess that method deserves a documentation now and I know you are good at it :)

Reply:
I had the same idea thought - that this needs extension to the documentation. And then I realized we have no documentation about main.pm at all :(

Reply:
So I would merge this without docu right now - but I first need to understand the change myself. So docu *would* help :)

Reply:
I actually don't mind documenting this once I have addressed @mudler's comments here: https://progress.opensuse.org/issues/27124